MekHQ/src/mekhq/campaign/CampaignController.java

```diff
@@ -57,11 +57,15 @@ public UUID getHost() {
     /**
      * Sets the unique identifier of the campaign hosting
      * this session.
-     * @param id The unique identifier of the host compaign.
+     * @param id The unique identifier of the host campaign.
      */
     public void setHost(UUID id) {
         host = id;
         isHost = getLocalCampaign().getId().equals(id);
+
+        // We need to change these based on which campaign is currently the host
+        // TODO: Add the following:
+        // something that returns CampaignGUI.showAdvanceMultipleDays(isHost);
```

Comment:
I don't understand this comment.

Reply:
This was me adding a TODO for however you decided to handle campaigns switching between being host or not, as this should be enabled on that switch.

Otherwise, we might want a getCampaignController Host current date or such.

Reply:
Removed based on that presumption.